src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  comments.map(comment => (
+    <div className="CommentList">
+      <CommentInfo comment={comment} key={comment.id} />
+    </div>
```

Comment:
```suggestion
    <div className="CommentList" key={comment.id}>
      <CommentInfo comment={comment} />
    </div>
```

move `key` for container